src/components/features/composite/FeatureWrapper.tsx

```diff
@@ -14,13 +15,18 @@ export const FeatureWrapper: FunctionComponent<PropsWithChildren<FeatureWrapperP
     const { children, feature, onRead } = props;
     const isColor = isColorFeature(feature);
     const isReadable = (feature.property && feature.access & FeatureAccessMode.ACCESS_READ) || isColor;
+
+    const parentFeature = props.parentFeatures?.[props.parentFeatures.length - 1]
+    let label = feature.label;
+    if (feature.name === 'state' && !['light', 'switch'].includes(parentFeature.type)) {
```

Comment:
@Koenkk this caused #1772 on devices with multiple endpoints. 

I think this is a missed check:
```suggestion
    if (parentFeature && feature.name === 'state' && !['light', 'switch'].includes(parentFeature.type)) {
```

Reply:
thanks, fixed!